Gaijinizer patch/Map107.txt

```diff
@@ -380,69 +389,72 @@ enshrined before you ...#MTLed
 > BEGIN STRING
 エコーについて
 > CONTEXT: Map107/events/5/pages/8/4/Choice/0
-About Echo#MTLed
+About Echo#Ron
 > END STRING
 
 > BEGIN STRING
 港への道について
 > CONTEXT: Map107/events/5/pages/8/4/Choice/1
-About the way to the port#MTLed
+The way to the port#Ron
 > END STRING
 
 > BEGIN STRING
 この森林の発生について
 > CONTEXT: Map107/events/5/pages/8/4/Choice/2
-About outbreak of this forest#MTLed
+This strange forest#Ron
 > END STRING
 
 > BEGIN STRING
 この場を去る
 > CONTEXT: Map107/events/5/pages/8/4/Choice/3
-Leave this place#MTLed
+Leave#Ron
 > END STRING
 
 > BEGIN STRING
 エコーは、ずっと一人でここにいるの？
 > CONTEXT: Map107/events/5/pages/8/6/Dialogue/kazuya
-Is Echo always here alone?#MTLed
+Are you always here by yourself?#Ron
 > END STRING
 
 > BEGIN STRING
 ううん、さっきも言ったけど、『エコー』は種族名。
 
 この水路に、私と同じ『エコー』がいるよ。
 > CONTEXT: Map107/events/5/pages/8/8/Dialogue/enemy13
-Yeah, I mentioned earlier, but "Echo" is a tribal
-name. In this channel there is the same 'Echo' as
-me.#MTLed
+Nope, like I said before, "Echo" is the name of
+my tribe. There are other "Echos" like me in this
+waterway.#Ron
 > END STRING
 
 > BEGIN STRING
 みんなあなたを警戒してるから、私が出てきたんだけど
 
 大丈夫、そのうちみんなも姿を現すと思うわ。
 > CONTEXT: Map107/events/5/pages/8/12/Dialogue/enemy13
-I'm out there because everyone is watching you
-Okay, I think everyone will come up with it.#MTLed
+I came out because everyone is cautious of you,
+but it's okay, I think everyone will come out
+before long.#Ron
```

Comment:
Minor flow:
_**I came out because everyone is cautious of you,
but once they know you're okay, I think everyone will
come out before long.**_